### PR TITLE
[frontend/backend] improve dashboard, pagination & filters perfs (#1580)

### PR DESCRIPTION
**Pull request by guillaumejparis**

### CODE REVIEW

**Comment by RomuDeuxfois** [comment]
We need to add on this PR the @LogExecutionTime on all endpoints handle by the team

**Comment by guillaumejparis** [comment]
Close #970

File: openbas-front/src/admin/components/simulations/simulation/Index.tsx [hunk 1]
```diff
@@ -164,12 +165,15 @@ const Index = () => {
   const exercise = useHelper((helper: ExercisesHelper) => helper.getExercise(exerciseId));
   useDataLoader(() => {
     setLoading(true);
-    dispatch(fetchExercise(exerciseId)).finally(() => setLoading(false));
+    dispatch(fetchExercise(exerciseId)).finally(() => {
+      setPristine(false);
```
Comment (RomuDeuxfois) [inline openbas-front/src/admin/components/simulations/simulation/Index.tsx:169]: Same here for pristive X loading

File: openbas-front/src/components/common/queryable/filter/useFiltersState.tsx [hunk 2]
```diff
@@ -74,7 +75,10 @@ const useFiltersState = (
   };
 
   useEffect(() => {
-    onChange?.(filtersState.filters);
+    if (hasBeenInitialized.current) {
```
Comment (RomuDeuxfois) [inline openbas-front/src/components/common/queryable/filter/useFiltersState.tsx:78]: Why this changes ?

File: openbas-front/src/components/common/queryable/pagination/PaginationComponentV2.tsx [hunk 3]
```diff
@@ -69,11 +69,6 @@ const PaginationComponentV2 = <T extends object>({
   const [options, setOptions] = useState<OptionPropertySchema[]>([]);
 
   useEffect(() => {
-    // Retrieve input from uri
-    if (queryableHelpers.uriHelpers) {
-      queryableHelpers.uriHelpers.retrieveFromUri();
```
Comment (RomuDeuxfois) [inline openbas-front/src/components/common/queryable/pagination/PaginationComponentV2.tsx:74]: Why removing this ?

File: openbas-front/src/components/common/queryable/pagination/usPaginationState.tsx [hunk 4]
```diff
@@ -19,7 +20,10 @@ const usPaginationState = (initSize?: number, onChange?: (page: number, size: nu
   };
 
   useEffect(() => {
-    onChange?.(page, size);
+    if (hasBeenInitialized.current) {
```
Comment (RomuDeuxfois) [inline openbas-front/src/components/common/queryable/pagination/usPaginationState.tsx:23]: Same here

**Comment by guillaumejparis** [comment]
Split into multiple PRs
